src/state/reducers/manifests.js

```diff
@@ -10,6 +10,8 @@ export const manifestsReducer = (state = {}, action) => {
       return {
         ...state,
         [action.manifestId]: {
+          ...state[action.manifestId],
+          ...action.properties,
```

Comment:
If `properties` is an object we got nested state. Is it possible to place the single properties directly in the manifest object?

Reply:
I'm not sure I understand the request or concern?

Reply:
Oh, sorry, I was wrong. The spread operator already does what I mean.